pkg/crc/machine/generate_bundle_linux.go

```diff
@@ -1,50 +1,25 @@
 package machine
 
 import (
-	"os"
+	"fmt"
 	"path/filepath"
 
 	"github.com/code-ready/crc/pkg/crc/constants"
 	crcos "github.com/code-ready/crc/pkg/os"
 )
 
-func copyDiskImage(destDir string, srcDir string) (string, string, error) {
-	tmpDir := filepath.Join(constants.MachineCacheDir, "image-rebase")
-	_ = os.RemoveAll(tmpDir) // clean up before using it
-	if err := os.Mkdir(tmpDir, 0700); err != nil {
-		return "", "", err
-	}
-	defer func() {
-		_ = os.RemoveAll(tmpDir) // clean up after using it
-	}()
+func copyDiskImage(destDir string) (string, string, error) {
+	const destFormat = "qcow2"
 
-	// We copy the overlay file because before doing the commit, we need to adjust the backing file.
-	baseImageName := filepath.Base(srcDir)
-	srcOverlayDiskPath := filepath.Join(constants.MachineInstanceDir, constants.DefaultName, baseImageName)
-	destOverlayDiskPath := filepath.Join(tmpDir, "overlayImage.qcow2")
-	if err := crcos.CopyFileContents(srcOverlayDiskPath, destOverlayDiskPath, 0644); err != nil {
-		return "", "", err
-	}
+	imageName := fmt.Sprintf("%s.qcow2", constants.DefaultName)
 
-	// We copy the base image because this is where we are going to commit the changes which were made to the VM
-	if err := crcos.CopyFileContents(srcDir,
-		filepath.Join(tmpDir, baseImageName), 0644); err != nil {
-		return "", "", err
-	}
+	srcPath := filepath.Join(constants.MachineInstanceDir, constants.DefaultName, imageName)
+	destPath := filepath.Join(destDir, imageName)
 
-	// Use qemu-img commands to rebase and commit it.
-	format := "qcow2"
-	if _, _, err := crcos.RunWithDefaultLocale("qemu-img", "rebase", "-F", format, "-b",
-		filepath.Join(tmpDir, baseImageName), destOverlayDiskPath); err != nil {
-		return "", "", err
-	}
-	if _, _, err := crcos.RunWithDefaultLocale("qemu-img", "commit", destOverlayDiskPath); err != nil {
+	_, _, err := crcos.RunWithDefaultLocale("qemu-img", "convert", "-f", "qcow2", "-O", destFormat, srcPath, destPath)
```

Comment:
With this change looks like it takes complete 30G space  and on my dev box where current method is working this is failed because of following error.
```
$ df -h
Filesystem      Size  Used Avail Use% Mounted on
devtmpfs         16G     0   16G   0% /dev
tmpfs            16G  1.7G   14G  11% /dev/shm
tmpfs           6.3G  2.9M  6.3G   1% /run
/dev/dm-2       147G   20G  120G  15% /
tmpfs            16G   40M   16G   1% /tmp
/dev/nvme0n1p2  976M  235M  675M  26% /boot
/dev/nvme0n1p1  200M   21M  180M  11% /boot/efi
/dev/dm-5       305G  194G   96G  67% /home
tmpfs           3.2G  316K  3.2G   1% /run/user/1000


DEBU (crc) DBG | time="2021-05-03T12:39:58+05:30" level=debug msg="Closing plugin on server side" 
DEBU Copying '/home/prkumar/.crc/cache/crc_libvirt_4.7.5/kubeconfig' to '/tmp/crc_custom_bundle416347381/crc_libvirt_4.7.5_1620025798/kubeconfig' 
DEBU Copying '/home/prkumar/.crc/machines/crc/id_ecdsa' to '/tmp/crc_custom_bundle416347381/crc_libvirt_4.7.5_1620025798/id_ecdsa_crc' 
DEBU Copying '/home/prkumar/.crc/cache/crc_libvirt_4.7.5/oc' to '/tmp/crc_custom_bundle416347381/crc_libvirt_4.7.5_1620025798/oc' 
INFO Copying the disk image to crc_libvirt_4.7.5_1620025798 
DEBU Absolute path of custom bundle directory: /tmp/crc_custom_bundle416347381/crc_libvirt_4.7.5_1620025798 
DEBU Running 'qemu-img convert -f qcow2 -O qcow2 /home/prkumar/.crc/machines/crc/crc.qcow2 /tmp/crc_custom_bundle416347381/crc_libvirt_4.7.5_1620025798/crc.qcow2' 
DEBU Command failed: exit status 1                
DEBU stdout:                                      
DEBU stderr: qemu-img: error while writing at byte 30857363456: No space left on device 
```

Reply:
In my testing, there was only a few kB difference in the generated image between the previous method (with `rebase`/`commit`/...) and the method using `qemu-img convert`. Is this some temporary space which is needed during the conversion process, or was the generated image 30GB, and significantly bigger than the image generated before this PR?